main/main.script

```diff
@@ -120,7 +120,7 @@ function init(self)
 		local iphonese = vmath.vector3(640, 1136, 1)
 		local umx = vmath.vector3(480, 854, 1)
 		
-		local using = tab8
+		local using = project2
```

Comment:
Do you want to keep this as the default?

Reply:
Yeah, it's the resolution that the app is designed with. But will change the name though, as they are not accurate.